docs/release-process.md

```diff
@@ -160,6 +162,9 @@ hub pull-request
 
 ### Propose promotion of artifacts
 
+The `k8s-container-image-promoter` tool is `cip` from [kubernetes-sigs/k8s-container-image-promoter](https://github.com/kubernetes-sigs/k8s-container-image-promoter).
+The `gsutil` tool may be obtained from `pip3`.
```

Comment:
We should also add information about `promobot-files`, but I haven't used that yet.

Reply:
Should we replace `k8s-container-image-promoter` with `cip`?

Reply:
Possibly. I'd like to hear what @justinsb thinks.

Reply:
Anyway, I have more changes pending. I'm trying to get the more straightforward ones in first.

Reply:
Fine by me, just meant that if the tool is `cip`, we may as well use that in commands below.